nupic/research/frameworks/dendrites/modules/dendritic_mlp.py

```diff
@@ -73,18 +75,19 @@ class DendriticMLP(nn.Module):
     """
 
     def __init__(
-        self, input_size, output_size, hidden_sizes, num_segments, dim_context,
-        kw, kw_percent_on=0.05, context_percent_on=1.0, dendrite_weight_sparsity=0.95,
-        weight_sparsity=0.95, weight_init="modified", dendrite_init="modified",
-        freeze_dendrites=False, output_nonlinearity=None,
-        dendritic_layer_class=AbsoluteMaxGatingDendriticLayer,
+            self, input_size, output_size, hidden_sizes, num_segments, dim_context,
+            kw, kw_percent_on=0.05, context_percent_on=1.0,
+            dendrite_weight_sparsity=0.95,
+            weight_sparsity=0.95, weight_init="modified", dendrite_init="modified",
+            freeze_dendrites=False, output_nonlinearity=None,
+            dendritic_layer_class=AbsoluteMaxGatingDendriticLayer,
```

Comment:
I think the indentation here is fine, esp since that's how [this stack overlow article](https://stackoverflow.com/questions/24988162/define-functions-with-too-many-arguments-to-abide-by-pep8-standard) does it.

Also, I copied this style from the [model utils file](https://github.com/numenta/nupic.research/blob/master/nupic/research/frameworks/pytorch/model_utils.py).

Reply:
> I left some minor comments. Before I approve, can you explain the rationale behind having multiple output heads and how you plan to use them? Is it so that you can index the one corresponding to the certain task during training, so that all other heads are unmodified?

Multiple output heads are used by the policy network, which outputs the mean and diagonal log std, each of which are separate heads, of a gaussian.